sdk/core/core-client/review/core-client.api.md

```diff
@@ -321,6 +321,7 @@ export interface SerializerOptions {
 export class ServiceClient {
     constructor(options?: ServiceClientOptions);
     readonly pipeline: Pipeline;
+    readonly scopes?: string[];
```

Comment:
The authenticate method requires user to pass in the `scope` - this is a way for the user to retrieve the `scope` from the client.